package.json

```diff
@@ -138,10 +140,14 @@
     "mocha": "^6.2.0",
     "mocha-jenkins-reporter": "^0.4.2",
     "mvn-artifact-download": "^4.0.0",
+    "ts-essentials": "^3.0.2",
     "tslint": "^5.20.0",
     "typescript": "^3.6.3",
     "vsce": "^1.66.0",
-    "vscode": "^1.1.36"
+    "vscode": "^1.1.36",
+    "vscode-extension-tester": "2.0.0",
```

Comment:
why not using the `^` like other version declaration?

Reply:
solved, uses same declaration